code/datums/components/hallucinations.dm

```diff
@@ -549,9 +600,138 @@ ABSTRACT_TYPE(/datum/component/hallucination)
 			for(var/mob/O in oviewers(world.view , my_target))
 				boutput(O, SPAN_ALERT("<B>[my_target] stumbles around.</B>"))
 
+/obj/fake_singulo
+	name = "gravitational singularity"
+	desc = "Perhaps the densest thing in existence, except for you."
+	icon = null
+	icon_state = null
+	density = 0
+	var/list/my_image_overrides = list()
+	var/list/eaten_atoms = list()
+	var/mob/my_target = null
+	var/spaget_count = 0
+	var/right_spinning = 0
+
+/obj/fake_singulo/New(var/loc, var/mob/target)
+	..()
+	SPAWN(60 SECONDS)
+		qdel(src)
```

Comment:
why hardcode this?

Reply:
you know, I meant it to be a parameter and I forgot to change it